cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java

```diff
@@ -60,10 +60,10 @@
   private boolean useBatchInLogCatchUp = true;
```

Comment:
Hi, the `catchUpTimeoutMS` is not consistent with the iotdb-cluster.properites, in the ClusterConfig.java, the value is 60 seconds, while in the iotdb-cluster.properites, the value is 600 seconds, may be we can change to 300 seconds as the snapshot may take a long time, how do you think?